src/libyang/Makefile

```diff
@@ -0,0 +1,23 @@
+.ONESHELL:
+SHELL = /bin/bash
+.SHELLFLAGS += -e
+
+MAIN_TARGET = $(LIBYANG)
+DERIVED_TARGETS = $(LIBYANG_DEV)
+DEPS = $(DEST)/$(SWIG_BASE) $(DEST)/$(SWIG)
+
+$(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+	rm -fr ./libyang-$(LIBYANG_VERSION)
+	sudo dpkg -i $(DEPS)
```

Comment:
I need an advice here. I don't know why, but I need to use sudo dpkg -i here, just dependency was not enough.

Reply:
you have a typo in your libyang rule file.